web/views/goal_view.ex

```diff
@@ -6,4 +6,8 @@ defmodule Healthlocker.GoalView do
     |> Earmark.as_html!
     |> raw
   end
+
+  def pretty_date(date) do
+    Date.to_iso8601(date)
```

Comment:
in-line comment here would be _really_ good to help the next person know why this date format was selected... 😉